src/test/java/org/prebid/server/bidder/improvedigital/ImprovedigitalBidderTest.java

```diff
@@ -536,9 +683,11 @@ private static BidRequest givenBidRequest(UnaryOperator<Imp.ImpBuilder> impCusto
     }
 
     private static Imp givenImp(UnaryOperator<Imp.ImpBuilder> impCustomizer) {
+        impCustomizer = Objects.nonNull(impCustomizer)
+                ? impCustomizer
+                : impBuilder -> impBuilder.video(Video.builder().build());
```

Comment:
Please, remove it. You can add a new method that will add video object by default.